dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/imports/bundle/persister/EventPersister.java

```diff
@@ -264,22 +266,15 @@ private void deleteDataValue(
 
   private void logDataValueChange(
```

Comment:
Should we rename this method to better reflect what its doing? I am assuming it does not only log data value changes? So something like `logEventChange()` ? or similar?